makePoplogCommander.sh

```diff
@@ -364,6 +549,14 @@ void * safe_malloc( size_t n ) {
     return ptr;
 }
 
+
+
+#else
+
+static_assert( false, "Not defined for operating systems other than Darwin nor Linux." );
```

Comment:
Why'd we have this apple conditional? I thought you said poplog doesn't work on macos?

Reply:
(I know this was already there in the code, you've just moved it)

Reply:
It was in the gist. Which is a terrible reason. It would be nice to get it working on MacOS but it'll take more than this lol. I will strip it out.